app/controllers/state_file/questions/az_public_school_contributions_controller.rb

```diff
@@ -69,16 +68,22 @@ def destroy
 
       private
 
+      def contributions
+        @contributions ||= current_intake.az322_contributions
+      end
+
+      def set_contribution_count = @contribution_count = contributions.count
+
       def az322_contribution_params
         params.require(:az322_contribution).permit(
-          :made_contribution,
           :school_name,
           :ctds_code,
           :district_name,
           :amount,
           :date_of_contribution_day,
           :date_of_contribution_month,
-          :date_of_contribution_year
+          :date_of_contribution_year,
+          state_file_az_intake_attributes: [:made_az322_contributions]
```

Comment:
I don't think this is a problem since we only use it to validate the initial form submission, but will intakes from before this change have their `made_az322_contributions` set to the default `:unfilled` despite potentially having 1 or more `az322_contributions` associated with them?

Reply:
And do you think we should do a data migration to keep prior intakes consistent with future ones?

Reply:
yes, i included a comment about backfilling the old intakes in that story you requested! do you see any issues with doing that as a follow-up (as opposed to as part of this story)?